SwiftAppTemplate/Screens/Authentication/AuthorizationScreen.swift

```diff
@@ -25,39 +16,27 @@ struct AuthorizationScreen: View {
     var body: some View {
         ZStack {
             VStack {
-                Header(text: getLocalString(title))
-                nameField
-                    .padding(.vertical, 20)
+                Header(text: "Login/SignUp")
```

Comment:
No localized string here in the header?

Reply:
text variable of Header view is type of LocalizedStringKey